test/makeProcessors.js

```diff
@@ -42,9 +42,9 @@ describe('processors', () => {
     assert(includes(extensions, '.graphql'));
   });
 
-  it('should escape backticks and prepend internalTag', () => {
-    const query = 'query { someValueWith` }';
-    const expected = 'ESLintPluginGraphQLFile`query { someValueWith\\` }`';
+  it('should wrap with backticks, escape properly and prepend internalTag', () => {
+    const query = 'query { search(q: "` \\n ${}") { title } }';
+    const expected = 'ESLintPluginGraphQLFile`query { search(q: "\\` \\\\n \\${}") { title } }`';
```

Comment:
as a general comment i think this addition would be clearer as its own test case — im having some trouble parsing what's actually being added here

Reply:
What I did here is:

- Updated the test description. This could have been done as a separate commit – the _first_ commit, since the updated test description is valid for the old test as well.
- Beefed up the test case with more stuff that needs escaping than backticks.

So I don't see how a separate test case would make sense :)

Reply:
just to double-check here - the `\\n` is getting escaped as `\\\\n` but is `\\n` the already escaped version of `\n`?

Reply:
Welcome to backslash hell :) I have dealt with this in several projects such as CoffeeScript and Prettier :)

The _real_ input is:

```
query { search(q: "` \n ${}") { title } }
```

and the _real_ output is:

```
ESLintPluginGraphQLFile`query { search(q: "\` \\n \${}") { title } }`
```